### PR TITLE
Apply auto-formatting as implemented in PR #10111

### PR DESCRIPTION
**Pull request by x3ro**
Let's get PR auto-formatting merged! 🎉  This replaces #10130 in the hopes of having a short discussion/review and then being able to merge!

Once this is merged I'd then push forward and get #10111 wrapped up, followed by a PR that integrates it into the build system.

🚂

### CODE REVIEW

File: tests/cortexm_common_ldscript/Makefile [hunk 1]
```diff
@@ -159,3 +134,37 @@ test-assert_overflow_%: $(ELFFILES_DEPS)
...
+  arduino-mkrfox1200 \
+  arduino-mkrzero \
+  blackpill \
+  blackpill-128kib \
+  bluepill \
+  bluepill-128kib \
+  feather-m0 \
+  opencm904 \
+  spark-core \
+  #
```
Comment (MrKevinWeiss) [inline tests/cortexm_common_ldscript/Makefile]: Maybe move the comment to the top and combine so we can use `:=`

**Comment by smlng** [comment]
so no need for reformatting

**Comment by smlng** [comment]
@x3ro, to me #10111 is still valid bc there will be corner-cases where we currently still need some kind of board list. What I wanted to point out here is, that before we reformat existing black/white lists we should look which of these can easily be replaced by using the feature required or blacklist method. 

Which means this PR is not completely invalid but not all changes are necessary or at least we can/should avoid changing them twice in short succession.

**Comment by x3ro** [comment]
>  at least we can/should avoid changing them twice in short succession.

I think I disagree with this, i.e. I don't see much of a problem with changing them in short succession, and there's always the risk that a supposedly easy change (e.g. removing the blacklists) takes longer than expected, thus in this case also delaying this here pull request.

That being said, I don't feel strongly about it and thus will leave this open until the relevant changes on the black- and whitelist side are done, so that I can update. Please ping me here in this issue once this is done @smlng or @aabadie or whoever is spearheading that task :)

**Comment by MrKevinWeiss** [comment]
I would like this done sooner than later.  It should be pretty simple clean everything at least.  Additional cleanups should benefit from working with cleaner makefiles.

Alternatively, we can just push the formatting script as a tool and not with the CI and let whoever wants to get the makefiles up to a standard with all the features grouped take care of it.  Then once everything fits the script invoke it with murdock.

**Comment by MrKevinWeiss** [comment]
ping

**Comment by smlng** [comment]
I think we need consensus here which way to go first, either this PR or postpone until board blacklists have been replaced by proper feature specs.

**Comment by x3ro** [comment]
What is the state on the proper feature specs?

**Comment by miri64** [comment]
> What is the state on the proper feature specs?

They are in. Care to rebase / rework this?

**Pull request closed without merging**